Gopkg.lock

```diff
@@ -204,6 +204,14 @@
   revision = "fd0aebc74e908868b09ac140fb5a53cb363884c1"
   version = "1.2"
 
+[[projects]]
+  branch = "master"
+  digest = "1:77e13fb4b86c85c80be600735851c209420438ed8d463f1d0d366d9a096ced83"
+  name = "github.com/gocarina/gocsv"
```

Comment:
If you're not using `gocsv` anymore, remove it from `lock` file.

Reply:
Thank you